README.md

```diff
@@ -332,7 +332,7 @@ _See also: [IT Asset Management]([Ralph](#it-asset-management))_
 * [jotgit](https://github.com/jdleesmiller/jotgit) - Git-backed real-time collaborative code editing.
 * [KDevelop](https://www.kdevelop.org/) - IDE by the people behind KDE.
 * [Light Table](http://lighttable.com/) - The next generation code editor.
-* [Lime](http://limetext.org/) - Aims to provide an open source solution to Sublime Text
+* [Lime](https://limetext.github.io/) - Aims to provide an open source solution to Sublime Text
```

Comment:
Actually it's not maintained anymore (https://github.com/limetext/lime - last commit Jan 2021 - see also https://github.com/limetext/lime/issues/597 and other issues)

Can you remove limetext completely?

Reply:
removed lime text